test/EndToEndTests/Common/Microsoft.OData.E2E.TestCommon/Common/Client/ODataModelRef/DeviceModelTerms.csdl

```diff
@@ -0,0 +1,14 @@
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
+  <edmx:Reference Uri="CoreTerms.csdl">
+    <edmx:Include Alias="Core" Namespace="Microsoft.OData.E2E.TestCommon.Common.Server.ODataModelRef.Core.V1"/>
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="Microsoft.OData.E2E.TestCommon.Common.Server.ODataModelRef.Vocabulary.V1" Alias="Self">

```

Comment:
There are some csdls with the same schema namespace. Why have them in different files?